src/routes/concepts/context.mdx

```diff
@@ -10,12 +10,11 @@ Context provides a way to pass data through the component tree without having to
 When you have a large [component tree](/concepts/components/basics#component-trees) that requires state to be shared, context can be used.
 Context can be employed to avoid [prop drilling](/concepts/components/props#prop-drilling), which is the practice of passing props through intermediate elements without using them directly.
 
-If you only want to avoid passing some props through a few levels, component composition is often a simpler solution than context.
+If you only want to avoid passing "some" props through a "few" layers, adjusting your component hierarchy may be an easier solution, when applicable.
```

Comment:
```suggestion
If you want to avoid passing some props through a few layers, when applicable, adjusting your component hierarchy may be an easier solution.
```